themes/hugo-tutorials/static/css/theme.css

```diff
@@ -181,7 +181,7 @@ html body {
   margin-bottom: 1rem;
 }
 .expandable.expanded .expandable-content {
-  max-height: 2300px;
+  max-height: 3000px;
```

Comment:
Do we need a hardcoded number here? Which element does it affect?

Let's investigate if we can use vh units or unlimited max-height

Reply:
It affecting in [/dgraph-1.1.0/intro/4/](https://tour.dgraph.io/dgraph-1.1.0/intro/4/)

Yes, I think adapting element's height to the child element's height with no height limit would be better.